src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -413,219 +417,225 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
 
   return (
     <>
-      <FormWrapper>
-        <Controller
-          name="assets"
-          control={control}
-          render={() => (
-            <MultiFileSelect
-              files={{
-                video: mediaAsset,
-                thumbnail: { ...thumbnailAsset, originalBlob: originalThumbnailAsset?.blob },
-              }}
-              onVideoChange={handleVideoFileChange}
-              onThumbnailChange={handleThumbnailFileChange}
-              editMode={isEdit}
-              error={fileSelectError}
-              onError={handleFileSelectError}
-              maxVideoSize={10 * 1024 * 1024 * 1024}
-            />
-          )}
-        />
-        <InputsContainer>
-          <StyledTitleArea
-            {...register(
-              'title',
-              textFieldValidation({ name: 'Video Title', minLength: 3, maxLength: 40, required: true })
+      <FormScrolling>
+        <FormWrapper>
+          <Controller
+            name="assets"
+            control={control}
+            render={() => (
+              <div ref={fileSelectContainerRef}>
+                <MultiFileSelectWrapper width={fileSelectContainerBounds.width}>
```

Comment:
Why is this needed?

Reply:
it's so the multi file selector sizes itself propperly as it now has to have position: fixed we need to defined it's width manually

Reply:
I've missed it's `fixed` before. Understandable then, but does it really need to be `fixed`? I did a quick experiment with `position: sticky; top: 32px` and it seems to be working fine without any manual width setting. Can you try that?

Reply:
sticky worked as well 👍